project/ws/app/src/lib/routes/app-toc/components/karmapoints-panel/karmapoints-panel.component.ts

```diff
@@ -13,7 +14,16 @@ export class KarmaPointsPanelComponent implements OnInit {
     @Output() clickClaimKarmaPoints = new EventEmitter<string>()
     kpData: any
 
-  constructor() { }
+  constructor(
+    private translate: TranslateService,
+    private langtranslations: MultilingualTranslationsService
+  ) {
+    if (localStorage.getItem('websiteLanguage')) {
+      this.translate.setDefaultLang('en')
```

Comment:
Can we use this piece of code from service file instead of writing here by creating a function